src/constants/specs.ts

```diff
@@ -1,4 +1,23 @@
-import VEGA_SPECS from '../../public/spec/vega/index.json';
-import VEGA_LITE_SPECS from '../../public/spec/vega-lite/index.json';
+import {getPublicPath} from '../utils/publicPath';
```

Comment:
Why is this needed? Instead can we load just like before in https://github.com/vega/editor/pull/1420/files?